code/datums/brain_damage/mild.dm

```diff
@@ -0,0 +1,240 @@
+	desc = "Patient experiences occasional bouts of muscle weakness."
+	scan_desc = "weak motor nerve signal"
+	gain_text = "<span class='warning'>Your muscles feel oddly faint.</span>"
+	lose_text = "<span class='notice'>You feel in control of your muscles again.</span>"
+
+/datum/brain_trauma/mild/muscle_weakness/on_life()
+	var/fall_chance = 5
+	if(owner.m_intent == "run")
+		fall_chance += 15
+	if(prob(fall_chance) && !owner.lying && !owner.buckled)
+		to_chat(owner, "<span class='warning'>Your leg gives out!</span>")
+		owner.Weaken(5)
+
+	else if(owner.get_active_hand())
+		var/drop_chance = 15
+		var/obj/item/I = owner.get_active_hand()
+		drop_chance += I.w_class
+		if(prob(drop_chance) && owner.drop_from_inventory(I))
+			to_chat(owner, "<span class='warning'>You drop [I]!</span>")
+
+	else if(prob(3))
+		to_chat(owner, "<span class='warning'>You feel a sudden weakness in your muscles!</span>")
+		owner.adjustHalLoss(25)
+	..()
+
+/datum/brain_trauma/mild/muscle_spasms
+	name = "Muscle Spasms"
+	desc = "Patient has occasional muscle spasms, causing them to move unintentionally."
+	scan_desc = "nervous fits"
+	gain_text = "<span class='warning'>Your muscles feel oddly faint.</span>"
+	lose_text = "<span class='notice'>You feel in control of your muscles again.</span>"
+
+/datum/brain_trauma/mild/muscle_spasms/on_life()
+	if(prob(25))
+		switch(rand(1,5))
+			if(1)
+				if(owner.canmove)
+					to_chat(owner, "<span class='warning'>Your leg spasms!</span>")
+					step(owner, pick(cardinal))
+			if(2)
+				if(owner.incapacitated())
+					return
+				var/obj/item/I = owner.get_active_hand()
+				if(I)
+					to_chat(owner, "<span class='warning'>Your fingers spasm!</span>")
+					I.attack_self(owner)
+			if(3)
+				var/prev_intent = owner.a_intent
+				owner.a_intent = I_HURT
+
+				var/range = 1
+				if(istype(owner.get_active_hand(), /obj/item/weapon/gun)) //get targets to shoot at
+					range = 7
+
+				var/list/mob/living/targets = list()
+				for(var/mob/M in oview(owner, range))
+					if(isliving(M))
+						targets += M
+				if(LAZYLEN(targets))
+					to_chat(owner, "<span class='warning'>Your arm spasms!</span>")
+					owner.ClickOn(pick(targets))
+				owner.a_intent = prev_intent
+			if(4)
+				var/prev_intent = owner.a_intent
+				owner.a_intent = I_HURT
+				var/list/mob/living/targets = list()
+				if(LAZYLEN(targets))
+					to_chat(owner, "<span class='warning'>Your arm spasms!</span>")
+					owner.ClickOn(owner)
+				owner.a_intent = prev_intent
+			if(5)
+				if(owner.incapacitated())
+					return
+				var/obj/item/I =owner.get_active_hand()
+				var/list/turf/targets = list()
+				for(var/turf/T in oview(owner, 3))
+					targets += T
+				if(LAZYLEN(targets) && I)
+					to_chat(owner, "<span class='warning'>Your arm spasms!</span>")
+					owner.throw_item(pick(targets))
+	..()
+
+/datum/brain_trauma/mild/nearsightedness
+	name = "Cerebral Near-Blindness"
+	desc = "Patient's brain is no loosely connected to its eyes."
```

Comment:
This sentence doesn't really make sense, Did you mean "now" loosely connected?

Reply:
Copy-paste error